integration_tests/integration/court-register-list.spec.ts

```diff
@@ -98,7 +98,53 @@ context('Court register - court list navigation', () => {
       expect(items[2].selected).to.equal(false)
     })
 
-    const pageResults = courtRegisterPagedPage.pageResults()
+    const pageResults = page.pageResults()
     pageResults.contains('Showing 1 to 3 of 4 results')
   })
+  it('Will display filter', () => {
+    // IndexPage.verifyOnPage().courtRegisterLink().click()  -  TODO Will need this when plumbing in the paged court list and removign the all courts list
+    IndexPage.verifyOnPage()
+    cy.visit('/court-register/paged') // TODO and this will need removing
+    const page = AllCourtsPagedPage.verifyOnPage()
+
+    page.showFilterButton().click()
+    page.mojFilter().should('be.visible')
+    page.hideFilterButton().click()
+    page.mojFilter().should('not.be.visible')
+  })
+  it('Will change the active filter', () => {
+    // IndexPage.verifyOnPage().courtRegisterLink().click()  -  TODO Will need this when plumbing in the paged court list and removign the all courts list
+    IndexPage.verifyOnPage()
+    cy.visit('/court-register/paged') // TODO and this will need removing
+    const page = AllCourtsPagedPage.verifyOnPage()
+
+    page.showFilterButton().click()
+    page.activeAllFilter().prev().should('have.attr', 'type', 'radio').should('be.checked')
+    page.activeOpenFilter().prev().should('have.attr', 'type', 'radio').should('not.be.checked')
+    page.activeClosedFilter().prev().should('have.attr', 'type', 'radio').should('not.be.checked')
+    page.activeOpenFilter().click()
+    page.applyFilterButton().click()
+    page.showFilterButton().click()
```

Comment:
It's not ideal that the filter is hidden upon reload and you have to open it again - will fix this in the next PR.